src/editpage/components/assistance/AssistanceBar.vue

```diff
@@ -86,12 +91,16 @@ export default {
           },
         },
       ],
+      searchQuery: '',
     };
   },
   methods: {
     query(command) {
-      const input = this.$refs.queryInput;
-      let query = input.value;
+      if (!this.hasQuery) {
+        return;
+      }
+
+      let query = this.searchQuery;
```

Comment:
```suggestion
      let query = this.searchQuery.trim();
```
I guess also here then